inst/shiny/colourGroupInput.R

```diff
@@ -0,0 +1,55 @@
+#'
+#'
+
+colourGroupInput = function(inputId) {
```

Comment:
@stc1 Is it possible to put these in the R/ directory as regular exported R functions? Or do they need to be with the shiny code?